common/src/main/java/com/wynntils/models/gear/type/GearType.java

```diff
@@ -5,34 +5,65 @@
 package com.wynntils.models.gear.type;
 
 import com.wynntils.models.character.type.ClassType;
+import java.util.List;
 import java.util.Locale;
 import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
 import net.minecraft.world.item.Items;
 
 public enum GearType {
     SPEAR(ClassType.Warrior, Items.IRON_SHOVEL, 0),
-    WAND(ClassType.Mage, Items.STICK, 0),
+    WAND(ClassType.Mage, Items.STICK, 0, List.of(Items.WOODEN_SHOVEL)),
     DAGGER(ClassType.Assassin, Items.SHEARS, 0),
     BOW(ClassType.Archer, Items.BOW, 0),
     RELIK(ClassType.Shaman, Items.STONE_SHOVEL, 7),
+    // FIXME: We need a complete mapping of damage values for ring, bracelet and necklace to be able
+    // to get rid of this (needed for crafted and unknown gear)
+    ACCESSORY(null, Items.FLINT_AND_STEEL, 0),
     RING(null, Items.FLINT_AND_STEEL, 2),
     BRACELET(null, Items.FLINT_AND_STEEL, 19),
     NECKLACE(null, Items.FLINT_AND_STEEL, 36),
-    HELMET(null, Items.LEATHER_HELMET, 0),
-    CHESTPLATE(null, Items.LEATHER_CHESTPLATE, 0),
-    LEGGINGS(null, Items.LEATHER_LEGGINGS, 0),
-    BOOTS(null, Items.LEATHER_BOOTS, 0),
+    HELMET(
+            null,
+            Items.LEATHER_HELMET,
+            0,
+            List.of(Items.CHAINMAIL_HELMET, Items.IRON_HELMET, Items.GOLDEN_HELMET, Items.DIAMOND_HELMET)),
+    CHESTPLATE(
+            null,
+            Items.LEATHER_CHESTPLATE,
+            0,
+            List.of(
+                    Items.CHAINMAIL_CHESTPLATE,
+                    Items.IRON_CHESTPLATE,
+                    Items.GOLDEN_CHESTPLATE,
+                    Items.DIAMOND_CHESTPLATE)),
+    LEGGINGS(
+            null,
+            Items.LEATHER_LEGGINGS,
+            0,
+            List.of(Items.CHAINMAIL_LEGGINGS, Items.IRON_LEGGINGS, Items.GOLDEN_LEGGINGS, Items.DIAMOND_LEGGINGS)),
+    BOOTS(
+            null,
+            Items.LEATHER_BOOTS,
+            0,
+            List.of(Items.CHAINMAIL_BOOTS, Items.IRON_BOOTS, Items.GOLDEN_BOOTS, Items.DIAMOND_BOOTS)),
     MASTERY_TOME(null, Items.ENCHANTED_BOOK, 0),
     CHARM(null, Items.CLAY_BALL, 0);
 
     private final ClassType classReq;
     private final Item defaultItem;
     private final int defaultDamage;
+    private final List<Item> otherItems;
```

Comment:
Well.. How well does this work with reskinned items? Did we ever fix the problem with reskinned items not parsing (which is arguably much more important than this IMO)

Reply:
I'm not sure what you mean that reskinned items do not parse? For me, normal items that are reskinned parse just fine.

And no, the combination of an unknown gear (that is not in Wynncraft's API) and reskinning will make it impossible for us to detect, since it changes type to diamond_shovel, regardless of what gear type it is... So then we are just shit out of luck.